framework/include/utils/Units.h

```diff
@@ -45,7 +45,7 @@ class MooseUnits
 public:
   MooseUnits();
   MooseUnits(const std::string & unit_string);
-  MooseUnits(Real f) : _factor(f), _base() {}
+  MooseUnits(Real f) : _factor(f), _shift(), _base() {}
```

Comment:
is this cross PR contamination?

Reply:
sure. I got tired of compilation warnings while working on this branch

Reply:
removed that commit and opened https://github.com/idaholab/moose/pull/20802

Reply:
ok. I thought this was from the coordinate transform PR actually, my bad